packages/extension-browser/webpack.config.js

```diff
@@ -58,3 +53,23 @@ module.exports = {
     },
     resolve: { alias: { url$: path.resolve(__dirname, 'dist/src/shims/url.js') } }
 };
+
+const contentScriptConfig = merge.smart(baseConfig, {
```

Comment:
Do we really need `merge.smart`? From what I'm reading in the [docs](https://www.npmjs.com/package/webpack-merge#smart-merging) it's more for the `loaders` stuff.
For our use case, isn't the following enough?

```ts
const contentScriptConfig = Object.assign({ }, baseConfig, {
    entry: { webhint: './dist/src/content-script/webhint.js' },
    output: { path: path.resolve(__dirname, 'dist') }
});
```

Reply:
Correct, its future proofing if the configs ever get conflicting loaders. Should I remove for now?

Reply:
If you don't mind yes. Unless we have an specific example already in mind of something we want to do relatively soon then let's remove it from here and package.json.

Thanks!